AuctionSite.Bl/User/Validators/PasswordValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AuctionSite.BL.Common;
+using AuctionSite.DataAccess;
+using AuctionSite.Shared;
+using AuctionSite.Shared.BindingModel;
+using AuctionSite.Shared.Enums;
+using AuctionSite.Shared.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace AuctionSite.BL.User.Validators
+{
+    public class PasswordValidator : IValidator<RegisterUserBindingModel>
```

Comment:
Na to też jest atrybut, do matchowania regex'em

Reply:
Jest :)